src/Evolution/Systems/GeneralizedHarmonic/BoundaryConditions/Bjorhus.hpp

```diff
@@ -0,0 +1,230 @@
+// Distributed under the MIT License.
+// See LICENSE.txt for details.
+
+#pragma once
+
+#include <array>
+#include <cstddef>
+#include <memory>
+#include <optional>
+#include <pup.h>
+#include <string>
+#include <type_traits>
+
+#include "DataStructures/DataBox/Prefixes.hpp"
+#include "DataStructures/DataVector.hpp"
+#include "DataStructures/Tensor/Tensor.hpp"
+#include "DataStructures/Variables.hpp"
+#include "Evolution/BoundaryConditions/Type.hpp"
+#include "Evolution/Systems/GeneralizedHarmonic/BoundaryConditions/BoundaryCondition.hpp"
+#include "Evolution/Systems/GeneralizedHarmonic/ConstraintDamping/Tags.hpp"
+#include "Evolution/Systems/GeneralizedHarmonic/Tags.hpp"
+#include "Options/Options.hpp"
+#include "Parallel/CharmPupable.hpp"
+#include "PointwiseFunctions/AnalyticData/Tags.hpp"
+#include "PointwiseFunctions/AnalyticSolutions/AnalyticSolution.hpp"
+#include "PointwiseFunctions/GeneralRelativity/Tags.hpp"
+#include "Time/Tags.hpp"
+#include "Utilities/Gsl.hpp"
+#include "Utilities/TMPL.hpp"
+
+/// \cond
+namespace domain::Tags {
+template <size_t Dim, typename Frame>
+struct Coordinates;
+}  // namespace domain::Tags
+/// \endcond
+
+namespace GeneralizedHarmonic::BoundaryConditions {
+namespace detail {
+enum class ConstraintPreservingBjorhusType {
+  ConstraintPreserving,
+  ConstraintPreservingPhysical
+};
+
+ConstraintPreservingBjorhusType
+convert_constraint_preserving_bjorhus_type_from_yaml(
+    const Options::Option& options);
+}  // namespace detail
+
+/*!
+ * \brief Sets constraint preserving boundary conditions using the Bjorhus
+ * method.
+ */
```

Comment:
It would be helpful to provide in this documentation a link to the Bjorhus paper that describes the general procedure, any available link to GR papers about the procedure applied to GH, and code references to the individual implementation functions that have all of the mathematical details for setting the respective components of the characteristic fields.

Reply:
Added a more descriptive summary with useful references.